test/built-ins/Atomics/waitAsync/bigint/good-views.js

```diff
@@ -45,11 +44,12 @@ $262.agent.start(`
 `);
 
 
-Promise.all([
-  $262.agent.getReportAsync(),
-  $262.agent.getReportAsync(),
-  $262.agent.getReportAsync(),
-]).then(outcomes => {
+(async () => {
+  const outcomes = [];
+
+  for (let i = 0; i < 3; i++) {
+    outcomes.push(await $262.agent.getReportAsync());
```

Comment:
_missed an opportunity to use async generators..._

To be honest this looks better anyway.